test/CustomsDeclaration.spec.js

```diff
@@ -41,51 +41,39 @@ describe('Customs Declaration Resource', function() {
 
     it('Sends the correct request', function() {
 
-      shippo.customsdeclaration.create({
-			"exporter_reference": "",
-			"importer_reference": "",
-			"contents_type": "MERCHANDISE",
-			"contents_explanation": "T-Shirt purchase",
-			"invoice": "#123123",
-			"license": "",
-			"certificate": "",
-			"notes": "",
-			"eel_pfc": "NOEEI_30_37_a",
-			"aes_itn": "",
-			"non_delivery_option": "ABANDON",
-			"certify": true,
-			"certify_signer": "Laura Behrens Wu",
-			"disclaimer": "",
-			"incoterm": "",
-			"items": [
-			  "0c1a723687164307bb2175972fbcd9ef"
-			],
-			"metadata": "Order ID #123123"
-      });
+      var payload = {
```

Comment:
nit: `var` vs. `let`?

Reply:
yeah, per `package.json` we only require node 0.12.18, which doesn't support `let`.  Also for consistency with the rest of the code.